checks.d/go_expvar.py

```diff
@@ -54,6 +54,7 @@ class GoExpvar(AgentCheck):
 
     def __init__(self, name, init_config, agentConfig, instances=None):
         AgentCheck.__init__(self, name, init_config, agentConfig, instances)
+        self._namespace = init_config.get('namespace', DEFAULT_METRIC_NAMESPACE)
```

Comment:
That sounds much more forward thinking than my implementation. I'll happily adjust to make it an instance param.